apps/web/lib/web/controllers/github_webhook_controller.ex

```diff
@@ -19,11 +20,13 @@ defmodule Web.GitHubWebhookController do
   end
 
   def process_issue(%{"issue" => issue, "repository" => %{"html_url" => url, "name" => name}}) do
-    {:ok, project} = Projects.get_or_insert(%{url: url, name: name})
-
-    issue
-    |> attributes(project)
-    |> Opportunities.insert_or_update
+    case Projects.get(%{url: url}) do
+      nil -> Logger.info("Received issue payload for untracked project: #{name} #{url}")
```

Comment:
We could potentially want this to be `Logger.error` since #33 would identify and discard untracked projects earlier in the request.  If we have a matching token but the url has changed someone should look into it.  Thoughts?

Reply:
I can certainly up the level. If it's `:warn` or beyond we'll want to add a `capture_log` so we aren't polluting the test output for expected behavior.

Reply:
@jeffweiss that'd be great!  I don't think we're using `capture_log` anywhere so this can serve as an example of how it should be used.